sky/packages/sky/lib/widgets/scaffold.dart

```diff
@@ -157,7 +156,11 @@ class RenderScaffold extends RenderBox {
       RenderBox box = _slots[slot];
       if (box != null) {
         assert(box.parentData is BoxParentData);
-        if ((box.parentData.position & box.size).contains(position)) {
+        // Use local coordinates for containment test so we apply transforms.
+        // But when we pass the position to box.hitTest, we only want to apply
+        // the top-level transform (box will apply its transforms).
+        Point local = box.globalToLocal(position);
```

Comment:
I need to apply the RenderBox transform, though. I've changed it to call box.applyPaintTransform instead.